frontend/packages/operator-lifecycle-manager/integration-tests-cypress/tests/create-namespace.spec.ts

```diff
@@ -0,0 +1,56 @@
+import { checkErrors, testName } from '../../../integration-tests-cypress/support';
+import { modal } from '../../../integration-tests-cypress/views/modal';
+
+describe('Create namespace from install operators', () => {
+  before(() => {
+    cy.login();
+    cy.createProject(testName);
+  });
+
+  afterEach(() => {
+    checkErrors();
+  });
+
+  after(() => {
+    cy.deleteProject(testName);
+    cy.logout();
+  });
+
+  const nsName = `${testName}-ns`;
+
+  it('creates namespace from operator install page', () => {
+    const operatorSelector = 'advanced-cluster-management-redhat-operators-openshift-marketplace';
+    cy.log('test namespace creation from dropdown');
+    cy.visit(`/operatorhub/ns/${testName}`);
+    cy.byTestID(operatorSelector).click();
+    cy.byLegacyTestID('operator-install-btn').click({ force: true });
+
+    // configure operator install
+    cy.byTestID('Select a namespace-radio-input').check();
+    cy.byTestID('dropdown-selectbox')
+      .click()
+      .wait(2000);
+    cy.byLegacyTestID('dropdown-menu')
+      .contains('Create Namespace')
+      .click();
+
+    // verify namespace modal is opened
+    modal.shouldBeOpened();
+    cy.byTestID('input-name').type(nsName);
+    modal.submit();
+    modal.shouldBeClosed();
+
+    cy.get('button')
+      .contains('Install')
+      .click();
+
+    // wait till the operator was successfully installed in the new namespace
+    cy.contains('Installed operator - ready for use', { timeout: 60000 });
+    cy.byTestID('view-installed-operators-btn').should(
+      'contain',
+      `View Installed Operators in namespace ${nsName}`,
+    );
```

Comment:
We really just need to make sure that the new ns was successfully created. No need to complete the full operator installation. We can just navigate to the project list page and make sure that the newly created project exists there, then delete it.

Reply:
removed waiting for operator to complete install.  Validating creation of new `ns` & `project` is already ensured by `cy.deleteProject(nsName)` on #L54 and #L15 respectively.